sway-lib-std/src/b512.sw

```diff
@@ -52,7 +52,27 @@ impl B512 {
     /// ```
     pub fn new() -> Self {
         Self {
-            bytes: [ZERO_B256, ZERO_B256],
+            bits: [ZERO_B256, ZERO_B256],
         }
     }
+
+    /// Returns the underlying bits for the B512 type.
+    ///
+    /// # Returns
+    ///
+    /// * [[b256; 2]] - The two `b256`s that make up the `B512`.
+    ///
+    /// # Examples
+    ///
+    /// ```sway
+    /// use std::{b512::B512, constants::ZERO_B256);
+    ///
+    /// fn foo() {
+    ///     let zero = B512::new();
+    ///     assert(zero.bits() == [ZERO_B256, ZERO_B256]);
```

Comment:
I dont think this compiles, theres no eq implemented for [b256;2]

Reply:
There is for [T; 2]